packages/map-template/src/components/MapsIndoorsMap/MapsIndoorsMap.jsx

```diff
@@ -153,6 +159,21 @@ function MapsIndoorsMap({ apiKey, gmApiKey, mapboxAccessToken, venue, locationId
     }, [appUserRoles]);
 
 
+    /**
+     * React on changes in the external id prop.
+     * Get the locations by external ids, if present.
+     */
+    useEffect(() => {
+        setExternalIdArray(externalIds);
+        if (externalIds && externalIdArray?.length > 0) {
```

Comment:
I am confused with us having both `externalIds` and `externalIdArray`.

What's the difference?

Reply:
Indeed there was no need for having both, I removed the state hook.